src/UserBundle/Maker/UserMaker.php

```diff
@@ -426,10 +426,17 @@ private function generateControllers(ConsoleStyle $io): void
 PHP;
         }
 
+        $wantsLogout = $io->confirm('Can users logout? (adds config/routes.yaml)');
```

Comment:
```
$hasLogout = $hasLogin && $io->confirm('...');
```

so it's only asked in case of login being available, whereas currently it's always asked but the actual write happens conditionally with `$hasLogin`